tutorial/set-name.md

```diff
@@ -115,12 +116,12 @@ func NewHandler(keeper Keeper) sdk.Handler {
 
 `NewHandler` is essentially a sub-router that directs messages coming into this module to the proper handler. At the moment, there is only one `Msg`/`Handler`.
 
-Now, you need to define the actual logic for handling the `MsgSetName` message in `handleMsgSetName`.:
+Now, you need to define the actual logic for handling the `MsgSetName` message in `handleMsgSetName`:
 
 > _*NOTE*_: The naming convention for handler names in the SDK is `handleMsg{ .Action }`
 
 ```go
-// Handle MsgSetName
+// Handle a message to set name
```

Comment:
If you are making a change in the code in the `.md` files, please make a corresponding change in the `.go` file.